sdks/python/apache_beam/runners/dataflow/internal/dependency.py

```diff
@@ -607,65 +151,9 @@ def get_sdk_name_and_version():
 def get_sdk_package_name():
```

Comment:
<!--new_thread; commit:6301a5b292792363810aa72bac875dc67c5c5fc9; resolved:0-->
Let's implement this in LegacyDataflowStager (see another comment).